docs/publications/BioHackrXiv/paper.md

```diff
@@ -0,0 +1,154 @@
+---
+title: 'How to increase the findability, visibility, and impact of Galaxy tools for your scientific community'
+title_short: 'BioHackEU23 project #25: Finding Galaxy tools'
+tags:
+  - Findability
+  - Galaxy
+  - Community specific
+  - Tools
+  - EDAM
+  - bio.tools
+  - Metadata
+authors:
+  - name: Paul Zierep*
+	orcid:
+	affiliation: 1
+  - name: Bérénice Batut
+	orcid: 0000-0001-9852-1987
+	affiliation: 1, 2
+  - name: Nicola Soranzo
+	orcid: 0000-0003-3627-5340
+	affiliation: 3
+  - name: Matúš Kalaš
+	orcid:
+	affiliation: 4
+  - name: Ove Johan Ragnar Gustafsson
+	orcid: 0000-0002-2977-5032
+	affiliation: 5
+affiliations:
+  - name: Bioinformatics Group, Department of Computer Science, University of Freiburg, Freiburg, Germany
+	index: 1
+  - name: Institut français de Bioinformatique, CNRS UAR 3601, Évry, France & Mésocentre Clermont-Auvergne, Université Clermont Auvergne, Aubiere, France
+	index: 2
+  - name: Earlham Institute, Norwich Research Park, Norwich, UK
+	index: 3
+  - name: 
+	index: 4
+  - name: Australian BioCommons, University of Melbourne, Melbourne, Victoria, Australia
+	index: 5
+date: 12 February 2024
+bibliography: paper.bib
+event: BioHackathon Europe 2023
+biohackathon_name: "BioHackathon Europe 2023"
+biohackathon_url:   "https://biohackathon-europe.org/"
+biohackathon_location: "Barcelona, Spain, 2023"
+group: BioHackrXiv
+git_url: https://github.com/galaxyproject/galaxy_tool_metadata_extractor/tree/e604ffd9866f9c32f797ceee45062b4d6f2a1513/docs/publications/BioHackrXiv
```

Comment:
Shouldn't this be a tree/main link instead?
And here we should consider whether we move it to the global biohackathon repo at submission time. In such case, the link should point there.
Do we know now whether it would interfere with JOSS publication or not, @bebatut ?

Reply:
Another idea (radical? but software-like😊):
Use only https://github.com/galaxyproject/galaxy_tool_metadata_extractor/tree/main/docs/publications/paper.md (without the BiohackrXiv), and update it to the next paper.md (for JOSS) afterwards. (And eventual subsequent publications after that)

Reply:
What do you think @bebatut & @paulzierep ?